src/cve/stages/convert_to_output_object.py

```diff
@@ -170,9 +194,11 @@ def convert_to_output_object(message: ControlMessage) -> AgentMorpheusOutput:
         if vuln_id in llm_engine_output:
             output.append(_parse_agent_morpheus_engine_output(llm_engine_output[vuln_id]))
         elif vuln_id in deficient_intel:
-            output.append(_get_deficient_intel_output(vuln_id))
+            output.append(_get_deficient_intel_output(llm_engine_output[vuln_id]))
         elif vuln_id in no_vulns:
-            output.append(_get_no_vuln_packages_output(vuln_id))
+            output.append(_get_no_vuln_packages_output(llm_engine_output[vuln_id]))
```

Comment:
@tmihalac Why did you change it? please remove.

Reply:
NVM I will revert it